Core/MIPS/IR/IRCompBranch.cpp

```diff
@@ -363,6 +368,9 @@ void IRJit::Comp_Syscall(MIPSOpcode op) {
 	RestoreRoundingMode();
 	js.downcountAmount = -offset;
 
+	int dcAmount = js.downcountAmount + 1;
```

Comment:
In this case there's no delay slot, and if we're already in a delay slot, it should be -1.  I think the other jits do this.

-[Unknown]